kythe/cxx/verifier/verifier.h

```diff
@@ -376,6 +382,12 @@ class Verifier {
 
   /// Identifier for MarkedSource false values.
   AstNode* marked_source_false_id_;
+
+  /// Maps from file content to (verified) VName.
+  std::unordered_map<Symbol, AstNode*> content_to_vname_;
```

Comment:
absl::flat_hash_map is a thing now and should be preferred (although we might have to update the absl dependency).